### PR TITLE
test(dsg): fix e2e test on generate apps

### PR DESCRIPTION
**Pull request by overbit**
Close: #5104 

## PR Details

This PR reintroduces some basic e2e tests against the generated app. 
It also fixes the issue about `.amplication` folder and the missing documentation reported here https://github.com/amplication/amplication/issues/5263
 
## PR Checklist

- [x] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: packages/data-service-generator/src/main.ts [hunk 1]
```diff
@@ -28,36 +23,14 @@ async function readInputJson(filePath: string): Promise<DSGResourceData> {
...
-      new URL(
-        "build-runner/code-generation-success",
-        process.env.BUILD_MANAGER_URL
-      ).href,
-      {
-        resourceId: process.env.RESOURCE_ID,
-        buildId: process.env.BUILD_ID,
-      }
-    );
+    await generateCodeByResourceData(resourceData, destination);
```
Comment (abrl91) [inline packages/data-service-generator/src/main.ts:33]: nice!

**Review by abrl91 — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -220,8 +220,10 @@
     "@typescript-eslint/eslint-plugin": "^5.39.0",
     "@typescript-eslint/parser": "^5.39.0",
     "babel-jest": "^28.1.0",
+    "base-64": "^1.0.0",
```
Comment (tupe12334) [inline package.json:223]: Don't we want to have the types? 
https://www.npmjs.com/package/@types/base-64

File: packages/data-service-generator/src/register-plugin.ts [hunk 3]
```diff
@@ -34,7 +35,7 @@ async function* getPluginFuncGenerator(
       yield func.default;
     } while (pluginListLength > index);
   } catch (error) {
-    console.log(error); /// log error
+    logger.log(error); /// log error
```
Comment (tupe12334) [inline packages/data-service-generator/src/register-plugin.ts:38]:
```suggestion
    logger.log(error);
```

**Review by tupe12334 — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (overbit) [reply package.json:223]: that is not necessary for now since it is used in only one line.

**Review by tupe12334 — APPROVED** [review]

File: packages/data-service-generator/src/create-data-service.ts [hunk 4]
```diff
@@ -53,7 +53,7 @@ export async function createDataService(
   } catch (error) {
     await createLog({
       level: "info",
```
Comment (yuval-hazaz) [inline packages/data-service-generator/src/create-data-service.ts:55]:
```suggestion
      level: "error",
```

**Review by yuval-hazaz — APPROVED** [review]
See minor comment.
Other than that, this is great and I would also like it to the next stage where we can create a matrix of different plugins and test different combination

**Pull request merged**